snuba/consumer.py

```diff
@@ -90,7 +96,7 @@ def flush_batch(self, batch: Sequence[ProcessedMessage]):
         if replacements:
             for key, replacement in replacements:
                 self.producer.produce(
-                    self.replacements_topic,
+                    self.replacements_topic.name,
```

Comment:
This technically introduces a new type error, but that's only because the constraints are stronger — this has always been problematic:

```snuba/consumer.py:99: error: Item "None" of "Optional[Topic]" has no attribute "name"```